nl/guides/for-students/courses/README.md

```diff
@@ -0,0 +1,155 @@
+---
+title: Cursus handleiding
+description: "Tutorial: cursus handleding"
+---
+
+# Cursus handleiding
+> Alle informatie zodat je als student kan registreren en uitschrijven voor een cursus, kan navigeren naar een cursus en de indienstatus kan interpreteren.
+
+## Navigeren naar een cursus
```

Comment:
This title seems to be wrong.
- The page should start with the explanation of a course
- The next section should be about registration (including how to search for courses and that courses you registered for are visible on the home page)
- Next section should be about the components of a course, series and deadline (much of that is in this section)
- Last section should be about unregistering